docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -60,8 +61,65 @@ to return the metadata pertaining to a partition or column respectively.
     and `col_name` are  mutually exclusive and can not be specified together. Currently
     nested columns are not allowed to be specified.
     
+    JSON format is not currently supported for individual columns.
+    
     **Syntax:** `[ database_name. ] [ table_name. ] column_name`
 
+* **AS JSON**
+
+  An optional parameter to return the table metadata in JSON format. Only supported when `EXTENDED`
+  or `FORMATTED` format is specified.
+
+  **Syntax:** `[ AS JSON ]`
+
+  **Schema:**
+  
+  Below is the full JSON schema. 
+  In practice only non-null fields are included in the JSON output (see Examples section).
+  
+    ```sql
+    {
+      "table_name": "<table_name>",
+      "catalog_name": "<catalog_name>",
+      "schema_name": "<schema_name>",
+      "namespaces": ["<schema_name>"],
+      "type": "<table_type>",
+      "provider": "<provider>",
+      "columns": [
+        {
+        "id": 1,
```

Comment:
JSON array is ordered, why do we need an extra id field to indicate the columns order?